<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PYTAKA</class>
 <widget class="QWidget" name="PYTAKA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>734</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1126</width>
    <height>734</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1126</width>
    <height>734</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="welcome_stacked">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1151</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(254, 255, 239);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="welcome_page">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QFrame" name="frame_butt">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>280</y>
       <width>201</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QPushButton" name="signup_button_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30px;
color: rgb(234, 236, 238);
background-color: rgb(24, 141, 143);</string>
      </property>
      <property name="text">
       <string>SIGN UP</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="login_button_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30px;
background-color: rgb(2, 56, 89);
color: rgb(234, 236, 238);
</string>
      </property>
      <property name="text">
       <string>LOGIN</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1131</width>
       <height>741</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/background/assets/background/welcome.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>frame</zorder>
    <zorder>frame_butt</zorder>
   </widget>
   <widget class="QWidget" name="sign_up">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 255, 239);</string>
    </property>
    <widget class="QFrame" name="signup_form">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>734</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(248, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="signup_butt">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>630</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30px;
background-color: rgb(2, 56, 89);
color: rgb(234, 236, 238);
</string>
      </property>
      <property name="text">
       <string>SIGN UP</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="first_name">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 20px;
	background-color: rgb(16, 89, 99);
	padding-top: 3 px;
	padding-left: 20 px;
	padding-right: 20 px;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>First Name</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="last_name">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 20px;
	background-color: rgb(16, 89, 99);
	padding-top: 3 px;
	padding-left: 20 px;
	padding-right: 20 px;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="placeholderText">
       <string>Last Name</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="date_of_birth">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>330</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 20px;
	background-color: rgb(16, 89, 99);
	padding-top: 3 px;
	padding-left: 20 px;
	padding-right: 20 px;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="placeholderText">
       <string>Date of Birth</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="email_add">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>430</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 20px;
	background-color: rgb(16, 89, 99);
	padding-top: 3 px;
	padding-left: 20 px;
	padding-right: 20 px;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="placeholderText">
       <string>Email Address</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>520</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 20px;
	background-color: rgb(16, 89, 99);
	padding-top: 3 px;
	padding-left: 20 px;
	padding-right: 20 px;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins ExtraBold</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Sign up here</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="prompt">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>590</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
color: rgb(0, 0, 0);
border: transparent;
</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Poppins'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>569</x>
       <y>-1</y>
       <width>581</width>
       <height>741</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/background/assets/background/sign-up.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="login">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 255, 239);
</string>
    </property>
    <widget class="QFrame" name="login_form">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>734</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(248, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins ExtraBold</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Welcome back!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login_butt">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>540</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30px;
background-color: rgb(2, 56, 89);
color: rgb(234, 236, 238);
</string>
      </property>
      <property name="text">
       <string>LOGIN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="log_email">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>300</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 20px;
	background-color: rgb(16, 89, 99);
	padding-top: 3 px;
	padding-left: 20 px;
	padding-right: 20 px;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="placeholderText">
       <string>Email Address</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="log_pass">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>390</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 20px;
	background-color: rgb(16, 89, 99);
	padding-top: 3 px;
	padding-left: 20 px;
	padding-right: 20 px;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="return_butt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins SemiBold</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(16, 89, 99);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="elements">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>0</y>
       <width>581</width>
       <height>734</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/background/assets/background/login_bg.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
